fusionauth/resource_system_configuration.go

```diff
@@ -187,6 +187,39 @@ func resourceSystemConfiguration() *schema.Resource {
 					},
 				},
 			},
+			"webhook_event_log_configuration": {
+				Type:       schema.TypeList,
+				MaxItems:   1,
+				Optional:   true,
+				Computed:   true,
+				ConfigMode: schema.SchemaConfigModeAttr,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"delete": {
+							Type:       schema.TypeList,
+							MaxItems:   1,
+							Optional:   true,
+							ConfigMode: schema.SchemaConfigModeAttr,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"enabled": {
```

Comment:
This will work, but there is also a `fusionauth.Enableable` struct that you may want to use to reduce boilerplate.

Reply:
I was modeling this after the way that audit log configuration does it. None of the other `Enableable`s use that struct in this `resourceSystemConfiguration()` function, but I did notice the new configuration was missing in `getDefaultSystemConfigurationRequest()` and added it there using `fusionauth.Enableable`.

Is `fusionauth.Enableable` usable in this function, and it's just not using that pattern? I admit I may need an example to reference for that update.

Reply:
I was able to test this locally. I think that `fusionauth.Enableable` is from the go client, but this portion is describing what it looks like in TF. I don't think it's possible to use that type here, but I was able to apply changes using this new configuration.